ExecuteStage/easyspider_executestage.py

```diff
@@ -248,59 +250,65 @@ def preprocess(self):
                 parameters["waitElementTime"] = 10
                 parameters["waitElementIframeIndex"] = 0
 
-            if node["option"] == 1:  # 打开网页操作
-                try:
-                    cookies = node["parameters"]["cookies"]
-                except:
-                    node["parameters"]["cookies"] = ""
-            elif node["option"] == 2:  # 点击操作
-                try:
-                    alertHandleType = node["parameters"]["alertHandleType"]
-                except:
-                    node["parameters"]["alertHandleType"] = 0
-                if node["parameters"]["useLoop"]:
+            if option == GraphOption.Get.value:  # 打开网页操作
+                cookies = parameters.get("cookies")
+                if not cookies:
+                    parameters["cookies"] = ""
+            elif option == GraphOption.Click.value:  # 点击操作
+                alertHandleType = parameters.get("alertHandleType")
+                if not alertHandleType:
+                    parameters["alertHandleType"] = 0
+                if parameters.get("useLoop"):
                     if self.task_version <= "0.3.5":
                         # 0.3.5及以下版本的EasySpider下的循环点击不支持相对XPath
-                        node["parameters"]["xpath"] = ""
-                        self.print_and_log("您的任务版本号为" + self.task_version +
-                                           "，循环点击不支持相对XPath写法，已自动切换为纯循环的XPath")
-            elif node["option"] == 3:  # 提取数据操作
-                node["parameters"]["recordASField"] = 0
-                try:
-                    params = node["parameters"]["params"]
-                except:
-                    node["parameters"]["params"] = node["parameters"]["paras"] # 兼容0.5.0及以下版本的EasySpider
-                    params = node["parameters"]["params"]
-                try:
-                    clear = node["parameters"]["clear"]
-                except:
-                    node["parameters"]["clear"] = 0
-                try:
-                    newLine = node["parameters"]["newLine"]
-                except:
-                    node["parameters"]["newLine"] = 1
+                        parameters["xpath"] = ""
+                        self.print_and_log(f"您的任务版本号为{self.task_version}，循环点击不支持相对XPath写法，已自动切换为纯循环的XPath")
+            elif option == GraphOption.Extract.value:  # 提取数据操作
+                parameters["recordASField"] = 0
+                params = parameters.get("params")
+                if not params:
+                    parameters["params"] = parameters["paras"]  # 兼容0.5.0及以下版本的EasySpider
+                    params = parameters["params"]
+
+                clear = parameters.get("clear")
+                if not clear:
+                    parameters["clear"] = 0
```

Comment:
Do you think this can be changed to `parameters["clear"] = parameters.get("clear", 0)` to make it more concise? Fits for all remaining parts.